tools/accuracy_checker/accuracy_checker/adapters/audio_recognition.py

```diff
@@ -0,0 +1,138 @@
+"""
+Copyright (c) 2019 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+from collections import defaultdict
+import string
+import numpy as np
+
+from ..adapters import Adapter
+from ..config import ConfigValidator, NumberField, BoolField, ConfigError
+from ..representation import CharacterRecognitionPrediction
+
+
+class CTCBeamSearchDecoder(Adapter):
+    __provider__ = 'ctc_beam_search_decoder'
```

Comment:
what the difference between your adapter and my beam_search_decoder?
https://github.com/opencv/open_model_zoo/blob/master/tools/accuracy_checker/accuracy_checker/adapters/text_recognition.py#L25

Reply:
I tried your implementation but output result differs from reference. This implementation of beam_search_decoder calculates result that match reference.